packages/playground/src/sidebar/runtime.ts

```diff
@@ -101,22 +165,13 @@ function rewireLoggingToElement(
       const eleContainerLog = eleOverflowLocator()
       allLogs.push(`${prefix}${output}<br>`);
       eleLog.innerHTML = allLogs.join("<hr />")
-      const scrollElement = eleContainerLog.parentElement
-      if (autoScroll && scrollElement) {
-        scrollToBottom(scrollElement)
+      if (autoScroll && eleContainerLog) {
+        eleContainerLog.scrollTop = eleContainerLog.scrollHeight
```

Comment:
Hah, well that's a drastic simplification - looks like mechanically it seems to be the same 👍🏻